Automattic/jetpack
# Charts: Adding line chart

Fixes #

## Proposed changes:

* adding basic version of a new line chart

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* run `pnpm i`
* `cd projects/js-packages/charts` and `pnpm run storybook`
* verify that Line chart loads properly in Storybook

<img width="1067" alt="SCR-20241205-oqdl" src="https://github.com/user-attachments/assets/35d04d7e-8668-489e-b345-388c803ccd39">
